numpy/testing/_private/utils.py

```diff
@@ -2654,8 +2653,7 @@ def _parse_size(size_str):
                 'kb': 1000, 'mb': 1000**2, 'gb': 1000**3, 'tb': 1000**4,
                 'kib': 1024, 'mib': 1024**2, 'gib': 1024**3, 'tib': 1024**4}
 
-    size_re = re.compile(r'^\s*(\d+|\d+\.\d+)\s*({0})\s*$'.format(
-        '|'.join(suffixes.keys())), re.I)
+    size_re = re.compile(fr'^\s*(\d+|\d+\.\d+)\s*({"|".join(suffixes.keys())})\s*$', re.I)
```

Comment:
Just a bit long.

Reply:
@charris no need to go over this with a fine-toothed comb. I'm going to make sure the line length gets auto-formatted correctly. I mistakenly thought the linter checked that, but E501 is disabled in our ruff config.